### ISSUE TITLE
Add Support for HACS Installation

### ISSUE THREAD

**Issue by rsampayo** [rsampayo/sleepme_thermostat#7]
Feature Request:

To enhance the accessibility and ease of use for the sleepme_thermostat integration, I plan to add support for installation via HACS (Home Assistant Community Store). This will allow users to discover, install, and update the integration more conveniently directly through HACS.

Proposed Actions:

	1.	HACS Compatibility:
	•	I will add a hacs.json file to the repository to ensure it meets the requirements for HACS compatibility.
	•	I will verify and adjust the repository structure as needed to align with HACS guidelines.
	2.	Register with HACS:
	•	I will submit the repository to the HACS default repository list, making the integration easier to find and install.

Benefits:

	•	Simplified Installation: Users will be able to install the integration with just a few clicks in HACS, reducing the need for manual setup.
	•	Automatic Updates: HACS will handle updates, ensuring users always have the latest version without manual intervention.
	•	Increased Visibility: Listing in HACS will increase the integration’s exposure within the Home Assistant community.

This enhancement will make the integration more user-friendly and accessible, improving the overall experience for current and future users.

**Comment by mikesalz** [comment]
Hey @rsampayo - Have you already started working on this one?  If not, I'm thinking about giving it a try.  If you don't mind.

**Comment by rsampayo** [comment]
> Hey @rsampayo - Have you already started working on this one? If not, I'm thinking about giving it a try. If you don't mind.

@mikesalz , Ive been thinking about asking you that for the past week. I was just waiting until I added the rest of the sensors. (which I just did https://github.com/rsampayo/sleepme_thermostat/releases/tag/v2.1.0) . So please go ahead! (and if you can please test out the new sensors.)

**Comment by mikesalz** [comment]
Hi @rsampayo - It is failing a bunch of HACS validations. But I did not expect it to work the first time. A dit probably won’t work the second or third time, but we’ll get there! I’ll review it later today.

**Comment by mikesalz** [comment]
Aldo, the new sensors look good so far

**Comment by rsampayo** [comment]
@mikesalz, feel free to add yourself to the manifest as codeowner.

**Comment by mikesalz** [comment]
@rsampayo I don't think we can expect the HACS PR to be approved any time soon.  I noticed there are tons of PRs going back months, which are just sitting there.  I guess the HACS folks are in no hurry to review them.  😩

**Comment by derekcentrico** [comment]
Any link to the PR?  Happy to join in to nag.

**Comment by krkeegan** [comment]
The PR is here: https://github.com/hacs/default/pull/2672

**Comment by derekcentrico** [comment]
Attempting again - https://github.com/hacs/default/pull/3643
